src/main/java/thrift/logic/commands/BudgetCommand.java

```diff
@@ -13,6 +13,17 @@
 
     public static final String COMMAND_WORD = "budget";
 
+    public static final String HELP_MESSAGE = COMMAND_WORD + ": Sets your monthly budget to the specified amount.\n"
+            + "Format: "
+            + COMMAND_WORD + " "
+            + CliSyntax.PREFIX_VALUE + "AMOUNT "
+            + CliSyntax.PREFIX_DATE + "MM/YYYY\n"
+            + "Possible usages of " + COMMAND_WORD + ": \n"
+            + "To set budget for this month (assuming this month is October 2019): "
+            + COMMAND_WORD + " " + CliSyntax.PREFIX_DATE + "500 " + CliSyntax.PREFIX_DATE + "10/2019\n"
+            + "To forecast your budget for upcoming months: "
```

Comment:
`"To forecast your budget for a particular month: "` <-- to imply that any month can be chosen, be it past or future